Prose/ProseLib/Sources/ProseCore/ProseClient.swift

```diff
@@ -65,11 +65,11 @@ public final class ProseClient: ProseClientProtocol {
 
   public func updateMessage(
     id: String,
-    newID: String,
+    carbonId: String,
```

Comment:
Please revert all instances where you've changed `newID` to `carbonId`. When you update a message you're not creating a message carbon but a new message. Only the server creates carbons. They are semantically different things.

Reply:
I don't agree with you because technically we're not updating the message ID, which is implied by `updateMessage`/`newID`. It was the reason for my initial change but I reverted because it's not important.

Reply:
I agree that the method signature is confusing. It's a leaky abstraction over XMPP details. `carbonId` would still be flat out wrong.